recipes/net-setSimultaneousAccepts-migration/src/workflow.ts

```diff
@@ -0,0 +1,309 @@
+import type { SgRoot, Edit, Range, SgNode } from '@codemod.com/jssg-types/main';
+import type Js from '@codemod.com/jssg-types/langs/javascript';
+
+/**
+ * Transform function that removes deprecated net._setSimultaneousAccepts() calls
+ *
+ * Handles:
+ * 1. net._setSimultaneousAccepts(true) → removed
+ * 2. net._setSimultaneousAccepts(false) → removed
+ * 3. Works with both CommonJS (require) and ESM (import) syntax
+ * 4. Handles aliased imports/requires
+ * 5. Removes unused variables/properties passed to the function
+ *
+ * Note: This was an internal API (DEP0121) that reached End-of-Life in Node.js v24.0.0
+ */
+export default function transform(root: SgRoot<Js>): string | null {
+	const rootNode = root.root();
+	const edits: Edit[] = [];
+	const linesToRemove: Range[] = [];
+
+	const netImportStatements = getAllNetImportStatements(root);
+	if (netImportStatements.length === 0) return null;
+
+	for (const statement of netImportStatements) {
+		processNetImportStatement(rootNode, statement, linesToRemove, edits);
+	}
+
+	if (edits.length === 0 && linesToRemove.length === 0) return null;
+
+	return applyTransformations(rootNode, edits, linesToRemove);
+}
+
+/**
+ * Collects all import/require statements for 'node:net'
+ */
+function getAllNetImportStatements(root: SgRoot<Js>): SgNode<Js>[] {
+	return [
+		...getNodeImportStatements(root, 'node:net'),
+		...getNodeImportCalls(root, 'node:net'),
+		...getNodeRequireCalls(root, 'node:net'),
+	];
+}
+
+/**
+ * Processes a single net import statement and finds all _setSimultaneousAccepts calls
+ */
+function processNetImportStatement(
+	rootNode: SgNode<Js>,
+	statementNode: SgNode<Js>,
+	linesToRemove: Range[],
+	edits: Edit[]
+): void {
+	const bindingPath = resolveBindingPath(statementNode, '$');
+	if (!bindingPath) return;
+
+	const callExpressions = findSetSimultaneousAcceptsCalls(rootNode, bindingPath);
+
+	for (const callNode of callExpressions) {
+		const argNode = callNode.getMatch('ARG');
+		
+		if (argNode) {
+			handleCallArgument(rootNode, argNode, linesToRemove);
+		}
+
+		removeCallExpression(callNode, linesToRemove, edits);
+	}
+}
+
+/**
+ * Finds all _setSimultaneousAccepts() call expressions
+ */
+function findSetSimultaneousAcceptsCalls(
+	rootNode: SgNode<Js>,
+	bindingPath: string
+): SgNode<Js>[] {
+	return rootNode.findAll({
+		rule: {
+			pattern: `${bindingPath}._setSimultaneousAccepts($ARG)`,
+		},
+	});
+}
+
+/**
+ * Handles the argument passed to _setSimultaneousAccepts()
+ * If it's a member expression or identifier that's only used here, marks it for removal
+ */
+function handleCallArgument(
+	rootNode: SgNode<Js>,
+	argNode: SgNode<Js>,
+	linesToRemove: Range[]
+): void {
+	const argKind = argNode.kind();
+
+	if (argKind === 'member_expression') {
+		handleMemberExpressionArgument(rootNode, argNode, linesToRemove);
+	} else if (argKind === 'identifier') {
+		handleIdentifierArgument(rootNode, argNode, linesToRemove);
+	}
```

Comment:
nit: switch makes it more clear that its inspecting the same condition for each.

```suggestion
  switch(argKind) {
    case 'member_expression': handleMemberExpressionArgument(rootNode, argNode, linesToRemove); break;
    case 'identifier': handleIdentifierArgument(rootNode, argNode, linesToRemove); break;
	}
```